modules/blocks/verify.js

```diff
@@ -221,7 +222,7 @@ __private.verifyAgainstLastNBlockIds = function(block, result) {
  * @returns {Array} result.errors - Array of validation errors
  */
 __private.verifyVersion = function(block, result) {
-	if (block.version > 0) {
+	if (!blockVersion.isValid(block.version, block.height)) {
```

Comment:
What's the path for Genesis Blocks with version 0?

Reply:
Genesis blocks are not going through `processBlock` (`modules/blocks/utils`) which calls `verifyBlock`, so the `version` is never checked for them.